sequence.go

```diff
@@ -92,3 +92,21 @@ func getFeatureSequence(feature Feature, location Location) string {
 
 	return sequenceString
 }
+
+//AnonymizeSequence takes a sequence struct and substitute identification data for a new anonymized string and returns this modified Sequence struct.
+func anonymizeSequence(sequence Sequence, sequence_id string) Sequence {
```

Comment:
capitalize first letter.

Reply:
Perfect. You think have an underscore in the params as sequence_id is a problem also?

Reply:
Yes. Golang style convention is camel case for almost everything except exported function.